extension/src/experiments/index.ts

```diff
@@ -177,6 +183,15 @@ export class Experiments extends BaseRepository<TableData> {
     return this.cliData.managedUpdate()
   }
 
+  public getMoreCommits() {
+    this.numberOfCommitsToShow = this.numberOfCommitsToShow + 2
+    void this.webviewMessages.changeHasMoreCommits(this.numberOfCommitsToShow)
```

Comment:
[C] Feels strange that `webviewMessages` is indirectly calling itself when this becomes an anonymous function.

Reply:
I agree. I might actually be able to move `numberOfCommitsToShow` into the webview messages. I'll try.

Reply:
Done. You're right, it feels better that way.